install.sh

```diff
@@ -88,3 +96,10 @@ wget https://github.com/mozilla/geckodriver/releases/download/v${GECKODRIVER_VER
 tar zxf geckodriver-v${GECKODRIVER_VERSION}-${GECKODRIVER_ARCH}.tar.gz
 rm geckodriver-v${GECKODRIVER_VERSION}-${GECKODRIVER_ARCH}.tar.gz
 mv geckodriver firefox-bin
+
+# Install node
+curl -sL https://deb.nodesource.com/setup_10.x | sudo bash -
```

Comment:
I noticed that old node or npm versions behave differently when it comes to for example this bug:

https://github.com/mozilla/openwpm-firefox-webext/issues/12

So maybe having always the latest version might help a lot.

Reply:
Yeah, the version of node is important, and different package managers seem to be tracking different versions (though I need to double check this).

I'd suggest we leave this in for now, and figure out a way to have OpenWPM depend on a specific version of node and handle that dependency gratefully (download and use a local copy instead of system-wide installation?)